CMakeLists.txt

```diff
@@ -23,10 +23,12 @@ link_directories(${PYTORCH_DIR}/lib)
 
 add_subdirectory(pybind11)
 add_subdirectory(${TVM_DIR})
+add_subdirectory(custom_tvm_ops)
```

Comment:
I think it's easier to just add a folder `torch_tvm/ops/` and add `${CMAKE_CURRENT_SOURCE_DIR}/torch_tvm/ops/*.cpp` after line 4.  This will avoid the need for new linked libraries with identical build files

Reply:
So I do not want the custom topi ops to be specific to torch_tvm. They can be used elsewhere as well if needed, so that was the main reason to keep the structure this way. It did introduce some issues but I think it is better this way. If you think otherwise let me know and we can resolve this.

Reply:
where else are these operators used?  should we just split it out into a separate repo then?

Reply:
Hmm. Yeah am not sure of that. Do you prefer to move this inside torch_tvm then and maybe move it out at a later point if we decide to?